tests/foreman/ui/test_errata.py

```diff
@@ -525,6 +527,49 @@ def test_positive_content_host_search_type(session, erratatype_vm):
         assert errata_ids == sorted(FAKE_9_YUM_SECURITY_ERRATUM)
 
 
+@pytest.mark.skip_if_open("BZ:1655130")
+@tier3
+def test_positive_content_host_errata_details(session, erratatype_vm, module_org, test_name):
+    """ Read for errata details on a content_host by user with only viewer permission
+
+    :id: 236de56f-8035-4072-b0fa-03abbf3fc692
+
+    :Steps:
+
+        1. Content Host with applicable security errata.
+        2. Create a User with 'Viewer' Permission.
+        3. Go to Content Hosts -> Select content host -> Errata Tab -> Select Errata_Id.
+
+    :expectedresults: The errata details apge should be disaplyed for User.
+
+    :BZ: 1655130
+
+    :CaseLevel: Integration
+    """
+    login = gen_string('alpha')
+    password = gen_string('alpha')
+    viewer_role = entities.Role().search(
+        query={'search': u'name="Viewer"'})
+    default_loc_id = entities.Location().search(
+         query={'search': 'name="{}"'.format(DEFAULT_LOC)})[0].id
+    default_loc = entities.Location(id=default_loc_id).read()
+
+    entities.User(role=viewer_role, login=login, password=password,
+                  default_location=default_loc, organization=[module_org]).create()
+
+    pkgs = " ".join(FAKE_9_YUM_OUTDATED_PACKAGES)
+    assert _install_client_package(erratatype_vm, pkgs, errata_applicability=True)
```

Comment:
I see the `pkgs` in only using at one place ..Can we use as below it :
```
assert _install_client_package(erratatype_vm, 
                               " ".join(FAKE_9_YUM_OUTDATED_PACKAGES), 
                               errata_applicability=True)
```

Reply:
In your above example, we are using three lines of code where an existing example uses the 2 lines of code. At looking at cleanness I think the current solution looks better.

Reply:
It was a suggestion only :) .. thanks